samples/scala-replicated-examples/src/main/proto/com/example/actions/double_counter.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+syntax = "proto3";
+package com.example.actions;
+
+import "akkaserverless/annotations.proto";
+
+import "google/protobuf/empty.proto";
+
+service DoubleCounter {
```

Comment:
for the records, this is here to make the codegen happy. We need at least one proto that is supported by the code gen. I'll remove this once we have codegen for replicated entities.